lambda/wirelessLogic/fetchWirelessLogicSIMDetails.spec.ts

```diff
@@ -30,6 +30,7 @@ void describe('getActiveSimsHistory()', () => {
 			apiKey: 'apiKey',
 			clientId: 'clientId',
 			wirelessLogicDataLimit: 5000000,
+			startMonth: 1,
```

Comment:
Is this January or February?

Reply:
January

Reply:
Consider using a constant here instead of a magic number.

Reply:
I changed the implementation since I made a mistake in the function itself. I'm now passing in a Date instead of a number so this should be good now!